Makefile

```diff
@@ -30,6 +30,7 @@ STATIC_BUILD          ?= true
 CI                    ?= false
 DB                    ?= postgres
 K3D                   := $(shell if [ "`kubectl config current-context`" = "k3s-default" ]; then echo true; else echo false; fi)
+ARGO_TOKEN            = $(shell kubectl -n argo get secret -o name | grep argo-server | xargs kubectl -n argo get -o jsonpath='{.data.token}' | base64 --decode)
```

Comment:
there are big differences between :=, ?= and = in Makefiles

Reply:
`kubectl -nargo get sa argo-server -o jsonpath='{.secrets[0].name}' | xargs kubectl -n argo get secret -o jsonpath='{.data.token}' | base64 --decode` might be more accurate, but this is okay.